function/services/application-mapper/application-question.js

```diff
@@ -122,9 +124,44 @@ function mapApplicationQuestion(data, applicationForm, addressDetails) {
 
             // Check if the applicant is eligible for special expenses
             case data.theme === 'special-expenses':
-                columnValue = applicationForm?.applicant_expenses ? 'true' : 'false';
+                columnValue = applicationForm?.applicant_expenses ? 'true' : data.value;
+                break;
```

Comment:
What would `data.value` be in the case of this being false?

Reply:
The logic is that if the applicant_expenses column value is true then keep it as true. If the column value is false or is not populated yet, then just get the raw value from the question set it to the column value.

It should remain as false unless one value is true, then it should remain as true.
Based on the JSON provided, the `data.value` can only ever be true or false:
```
{
            "type": "theme",
            "id": "special-expenses",
            "title": "Special expenses",
            "values": [
                {
                    "id": "q-applicant-se-treatment",
                    "type": "simple",
                    "label": "NHS or other state health service treatment",
                    "value": false,
                    "valueLabel": "No",
                    "sectionId": "p-applicant-se-treatment",
                    "theme": "special-expenses"
                },
                {
                    "id": "q-applicant-se-home-care",
                    "type": "simple",
                    "label": "Care or supervision",
                    "value": false,
                    "valueLabel": "No",
                    "sectionId": "p-applicant-se-home-care",
                    "theme": "special-expenses"
                },
                {
                    "id": "q-applicant-se-home-changes",
                    "type": "simple",
                    "label": "Changes to the home",
                    "value": false,
                    "valueLabel": "No",
                    "sectionId": "p-applicant-se-home-changes",
                    "theme": "special-expenses"
                },
                {
                    "id": "q-applicant-se-equipment",
                    "type": "simple",
                    "label": "Special equipment",
                    "value": false,
                    "valueLabel": "No",
                    "sectionId": "p-applicant-se-equipment",
                    "theme": "special-expenses"
                },
                {
                    "id": "q-applicant-se-aids",
                    "type": "simple",
                    "label": "Physical aids",
                    "value": false,
                    "valueLabel": "No",
                    "sectionId": "p-applicant-se-aids",
                    "theme": "special-expenses"
                },
                {
                    "id": "q-applicant-se-other",
                    "type": "simple",
                    "label": "Any other expenses or future expenses",
                    "value": false,
                    "valueLabel": "No",
                    "sectionId": "p-applicant-se-other",
                    "theme": "special-expenses"
                }
            ]
        },
```

Reply:
My only concern about this is mixing the types for boolean questions. It's either going to be string true or boolean false? That would make it the only boolean value in this dataset as everything else will get converted by the catch all in the default. I'd consider making the false value be 'N' for consistency? If it's always been like this then ignore.